lmdeploy/serve/openai/api_server.py

```diff
@@ -1054,13 +1054,15 @@ def serve(model_path: str,
 
     _, pipeline_class = get_task(model_path)
 
+    vision_config = VisionConfig(kwargs.get("vision_max_batch_size", 1))
```

Comment:
we already set it in https://github.com/InternLM/lmdeploy/blob/v0.6.1/lmdeploy/cli/utils.py#L476